Northstar.Client/mod/resource/northstar_client_localisation_english.txt

```diff
@@ -321,5 +321,10 @@ Press Yes if you agree to this. This choice can be changed in the mods menu at a
 		"INVALID_MASTERSERVER_TOKEN" 	"Invalid or expired masterserver token"
 		"JSON_PARSE_ERROR" 				"Error parsing json response"
 		"UNSUPPORTED_VERSION" 			"The version you are using is no longer supported"
+
+		// Incorrect Mod Installation
+		"INCORRECT_MOD_INSTALL_TOAST"		"Some of your mods are incorrectly installed. Click here for more info."
+		"INCORRECT_MODS_MODLIST_HEADER"		"Incorrectly Installed Mods (%s1)"
+		"INCORRECT_MOD_INSTALL_DESCRIPTION"	"This mod is incorrectly installed. Every mod has a file called \"mod.json\" in its root directory, however for this mod it was not found.\n\nExpected location: %s1\nFound location: %s2\n\nMake sure both mod.json file and mod directory are direct children of your mod folder."
```

Comment:
`mod.json` should be mentioned in both expected and found locations.

Reply:
resolved in 093c0ee